pkgs/applications/networking/cluster/helmsman/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildGoModule, fetchFromGitHub, ... }:
+
+let version = "3.3.0"; in
```

Comment:
This is somewhat unusual. You can add `rec` after `buildGoModule` and replace the `inherit` with a `version` definition.

Reply:
Thank you for your review 😊